perf/src/index.tsx

```diff
@@ -1,3 +1,5 @@
+import '@babel/polyfill'
```

Comment:
could you, please, suggest why this change was necessary? where it is used?

Reply:
It contains `regeneratorRuntime` that it's required by `async/await` transform.